doc/dnf5_plugins/needs_restarting.8.rst

```diff
@@ -20,14 +20,14 @@
 
 .. _needs_restarting_plugin_ref-label:
 
-#########################
- Needs-Restarting Command
-#########################
+########################
+needs-restarting Command
+########################
 
 Synopsis
 ========
 
-``dnf5 needs-restarting [--services | -s]``
+``dnf5 needs-restarting [-p | --processes [-e | --exclude-services]] [-s | --services] [-r | --reboothint]``
```

Comment:
`-r` was not documented before? 👀

Reply:
It was not as far as I could tell.  It was a no-op, so that may have been deliberate, but even in that case I think it should be documented both in the --help output and the man page since that is common for command line tools still carrying no-op options for backwards compatibility (e.g., see anything in GNU coreutils).